### PR TITLE
Update Client SDK docs for recent changes

### PR DESCRIPTION
**Pull request by prateekcr**
Signed-off-by: Chandrakar, Prateek <prateek.chandrakar@intel.com>

### CODE REVIEW

**Review by trbehera — APPROVED** [review]

**Review by KiranSukhavasi — APPROVED** [review]

File: docs/client-sdk/client-sdk-porting-guide.md [hunk 1]
```diff
@@ -146,15 +140,16 @@ The purpose of these defines is to specify the location where the reference solu
 !!! note
     These flags are not necessary for the platforms which have their own Secure Storage mechanisms. The platform may be able to store all blobs using Authenticated Encryption including Normal.blob. Client SDK always uses `fdo_blob_read()` to read the data, so, the underlying detail is already abstracted. In the reference solution, it is expected that these files exist physically although without any content. The content gets generated on an as-needed basis.
 
-##### MANUFACTURER_(IP/DN/PORT)
-Client SDK uses the location defined by the below flags to connect to Manufacturer Server to perform Device Initialization.
+##### MANUFACTURER_ADDR
+Client SDK uses the location defined in the specified file to connect to Manufacturer Server to perform Device Initialization. The format for the Manufacturer Address is of the form: `{http,https}://{DNS,IP}:port`. The following rules apply while setting the value and all of these are mandatory:
```
Comment (v-vasavi) [inline docs/client-sdk/client-sdk-porting-guide.md:144]: specified file - which file?

File: docs/client-sdk/client-sdk-porting-guide.md [hunk 2]
```diff
@@ -146,15 +140,16 @@ The purpose of these defines is to specify the location where the reference solu
...
 
-* **MANUFACTURER_IP:** This is the manufacturer IP address to be used for performing Device Initialization.
-* **MANUFACTURER_DN:** This is the manufacturer Domain Name used to resolve to the IP address. This is used if the MANUFACTURER_IP is not given.
-* **MANUFACTURER_PORT:** This is the manufacturer server port, where the Device Initialization protocol is waiting for the device. By default, the port is 8039 if this blob is not created, otherwise, the port is used from this blob.
+* The transport protocol value must be either http or https (case-sensitive). Any other value will result in an error.
+* Either one of DNS or IP Address can be provided. The maximum value of DNS is 100 characters and must only contain alphanumeric characters (0-9A-za-z), hyphens (-) and dot (.).
+* The port should be an integer between (1-65535).
+* The URL separators :// and : should be present at appropriate indices as per the indices of the above values.
```
Comment (v-vasavi) [inline docs/client-sdk/client-sdk-porting-guide.md:149]: would it be good to give an example here?

File: docs/client-sdk/client-sdk-porting-guide.md [hunk 3]
```diff
@@ -181,45 +176,22 @@ This cmake file consists mostly of build configuration from crypto.conf but is n
...
-SUPPORTED_AES_MODE = cbc ctr
+SUPPORTED_AES_MODE = gcm ccm
 ```
 
-  * **SUPPORTED_KEX:** This specifies the key exchange algorithms implemented to create a secure channel between Device and Owner in Ownership Transfer Protocol.
-
-    * ecdhxxx: Based on Elliptic Curve Diffie Hellman
-
-  * **SUPPORTED_DA:** This specifies the supported Device Attestation algorithms which device uses to prove its identity to Rendezvous Server and Owner.
+  * **SUPPORTED_DA:** This specifies the supported Device Attestation algorithms which device uses to prove its identity to Rendezvous Server and Owner. This automatically configures the Key Exchange algorithm (ECDH256/ECDH384) that is used to generate the shared secret and th AES Mode, to use higher crypto in the source.
```
Comment (v-vasavi) [inline docs/client-sdk/client-sdk-porting-guide.md:186]: and th AES mode - e missing in "the "

File: docs/client-sdk/client-sdk-porting-guide.md [hunk 4]
```diff
@@ -280,15 +252,15 @@ This function initializes the Client SDK data structures. It allows the applicat
 !!! note
     The reference application allows the Client SDK to retry for 5 times before calling abort
 
-The Client SDK allows the owner to download the required Device Management System agents via Service Info mechanism; the num_modules and module_information registers the Service Info modules with the Client SDK.
+The Client SDK allows the owner to download the required Device Management System agents via ServiceInfo mechanism; the num_modules and module_information registers the ServiceInfo modules with the Client SDK.
```
Comment (v-vasavi) [inline docs/client-sdk/client-sdk-porting-guide.md:255]: num_modules and mod_information registers or register ?

File: docs/client-sdk/client-sdk-reference-guide.md [hunk 5]
```diff
@@ -119,20 +119,20 @@ If an error occurs after a module has been initialized, during the remainder of
...
+The Device initally sends the 'devmod' module in the first round. Other module is sent to the Owner in subsequent rounds. As mentioned previously, only one Device ServiceInfo module is supported currently, i.e, `devmod` module, which is configured statically in `add_module_devmod()`. All but one parameters are configured in the same method, except Device ServiceInfo Key `devmod:modules` that is partially configured here, and in `fdo_serviceinfo_modules_list_write()`. The total number of supported Owner ServiceInfo modules and their names must be updated in the above mentioned methods. Support for handling multiple Device ServiceInfo modules will be added in future releases.
 
-Note that Device ServiceInfo is one way – from device to Owner Server. The Owner Server cannot respond to any Device ServiceInfo message during this phase.
+Note that Device ServiceInfo is one way for now – from device to Owner Server. The Owner Server cannot respond to any Device ServiceInfo message during this phase.
```
Comment (v-vasavi) [inline docs/client-sdk/client-sdk-reference-guide.md:124]: one way - should it be hyphenated as one word?

**Pull request merged**